tests/functional/cylc-config/09-platforms.t

```diff
@@ -20,7 +20,7 @@
 #-------------------------------------------------------------------------------
 set_test_number 4
 #-------------------------------------------------------------------------------
-cat > "${PWD}/global.cylc" <<__HEREDOC__
+cat > "global.cylc" <<__HEREDOC__
```

Comment:
Use the standard function for this (`create_test_global_config` or something like that).